config/crd/bases/OWNERS

```diff
@@ -4,4 +4,6 @@ filters:
   "^.*rosa.*\\.yaml$":
     approvers:
       - muraee
-      - stevekuznetsov
+      - serngawy
+    reviewers:
```

Comment:
Being an approver also grants reviewer status. Do you want to remove the duplication?

Alternatively, if we have some other ROSA devs, they could be added here.